.github/workflows/ci.yml

```diff
@@ -95,3 +96,82 @@ jobs:
 
       - name: Run specs
         run: npm run sass-spec -- --impl libsass -c ../sassc/bin/sassc
+
+  dart_sass_js:
+    name: "JS API | Dart Sass | ${{ matrix.os }}"
+    runs-on: ubuntu-latest
+    if: "github.event_name != 'pull_request' || !contains(github.event.pull_request.body, 'skip dart-sass')"
```

Comment:
should this be a different skipping tag than for the specs ? A PR will probably need to skip either the JS API test or the spec, depending on what it changes.

Reply:
Most of the time, I expect we'll be updating *either* the language *or* the JS API, so skipping the other one won't matter either way. Even if we're updating both, though, they'll still be paired with one Dart Sass PR that implements both the language and JS API changes, and in that case we will want to skip both.

Plus, this was is much terser :grinning:.